electrum/lnwire/peer_wire.csv

```diff
@@ -180,35 +185,35 @@ msgdata,channel_update,cltv_expiry_delta,u16,
 msgdata,channel_update,htlc_minimum_msat,u64,
 msgdata,channel_update,fee_base_msat,u32,
 msgdata,channel_update,fee_proportional_millionths,u32,
-msgdata,channel_update,htlc_maximum_msat,u64,,option_channel_htlc_max
+msgdata,channel_update,htlc_maximum_msat,u64,
```

Comment:
I am seeing some `UnexpectedEndOfStream` exceptions and lnpeer-disconnections, apparently due to this. Added better logging in https://github.com/spesmilo/electrum/commit/ab953f4a7f9add8fa87c71795c36b851685b75b0, e.g.:
```
28.51 | I | P/lnpeer.Peer.[LNGossip, 022d509049-f58b9aca] | failed to parse message from peer. disconnecting. msg_type=channel_update(258). exc=UnexpectedEndOfStream()
```

Reply:
Right. I guess these are old nodes that still do not send the mandatory field.
Since raw messages are saved in the db, these old messages might also crash the SQL thread when we parse them in `channel_db.load_data`.
```
 19.33 | E | asyncio | Future exception was never retrieved
future: <Future finished exception=UnexpectedEndOfStream()>
Traceback (most recent call last):
  File "/opt/electrum/electrum/sql_db.py", line 56, in run_sql
    result = func(self, *args, **kwargs)
  File "/opt/electrum/electrum/util.py", line 446, in <lambda>
    return lambda *args, **kw_args: do_profile(args, kw_args)
  File "/opt/electrum/electrum/util.py", line 442, in do_profile
    o = func(*args, **kw_args)
  File "/opt/electrum/electrum/channel_db.py", line 739, in load_data
    p = Policy.from_raw_msg(key, msg)
  File "/opt/electrum/electrum/channel_db.py", line 126, in from_raw_msg
    payload = decode_msg(raw)[1]
  File "/opt/electrum/electrum/lnmsg.py", line 511, in decode_msg
    parsed[field_name] = _read_field(fd=fd,
  File "/opt/electrum/electrum/lnmsg.py", line 117, in _read_field
    raise UnexpectedEndOfStream()

```

Reply:
I added a commit that prevents `load_data` from crashing on old messages: 89df68d66a739e4e4af9fb9b5c80f8d8a85b31a0